common/log/tag/tags.go

```diff
@@ -267,11 +267,16 @@ func MetricScope(metricScope int64) Tag {
 	return newIntegerTag("metric-scope", metricScope)
 }
 
+// StoreType returns tag for StoreType
+func StoreType(storeType string) Tag {
+	return newPredefinedStringTag("store-type", storeType)
+}
+
 // history engine shard
 
 // ShardID returns tag for ShardID
-func ShardID(shardID int64) Tag {
-	return newIntegerTag("shard-id", shardID)
+func ShardID(shardID int) Tag {
+	return newIntegerTag("shard-id", int64(shardID))
```

Comment:
I will create new helpers: newInt32 newInt64 to replace newIntegerTag so that we don't have to convert it.